### PR TITLE
Slim down base image

### PR DESCRIPTION
**Pull request by cjh1**

### CODE REVIEW

**Comment by cjh1** [comment]
This cuts things down a bit ..

File: docker/oremda/Dockerfile [hunk 1]
```diff
@@ -1,8 +1,16 @@
...
+RUN apt-get update && \
+  apt-get install -y \
+    git \
+    gcc && \
+  rm -rf /var/lib/apt/lists/*
 
 COPY . /oremda
 
-RUN pip install -e /oremda
+RUN pip install /oremda && \
```
Comment (alesgenova) [inline docker/oremda/Dockerfile:11]: I'm not 100% sure, but I think `-e` might be needed in order for the bind mount of the oremda dir during development

**Review by alesgenova — COMMENTED** [review]
Nice, how much space do we end up saving?

**Comment by cjh1** [comment]
Cuts it in half, I can rework the pip install to preserve `-e`

File: docker/oremda/Dockerfile (hunk 1 above)
Comment (cjh1) [reply docker/oremda/Dockerfile]: Fixed

**Pull request merged**